doc/arkode/guide/source/Usage/MRIStep/User_callable.rst

```diff
@@ -830,20 +767,14 @@ Optional inputs for IVP method selection
 
    Select the default MRI method of a given order.
 
-   The default order is 3. An order less than 1 or greater than 4 will result in
+   The default order is 3. An order less than 1 will result in
```

Comment:
No, but since MRIStep supports methods of different orders for different integration types (adaptive, fixed, explicit, ImEx, etc.), this routine is not the appropriate location to do this kind of error-checking.  Instead, if a user requests an order that is not supported, it will get caught inside `mriStep_SetCoupling` and return to the user with an error.